flake.nix

```diff
@@ -30,6 +30,8 @@
             pkgs.kubernetes-helm
             pkgs.nodejs
             pkgs.openshift
+            pkgs.python3
+            pkgs.python3Packages.pip
```

Comment:
I believe this is unnecessary and does not work as you would expect. In essence this creates another python instance inside the nix store (one without pip and one with pip inside its site packages). It also seems to interfere when creating venvs.

I think if you just keep `pkgs.python3` without any packages, the normal venv workflow should work. However, ONLY the venv workflow will work, as installing normal site-packages will not be possible. Not sure if this causes problems if you keep such a python version first in PATH when sourcing in `.bash_profile`.

Reply:
You were right, I think the step I missed was forgetting to delete and re-create `.venv` after installing python via nix.